kivakit-core/src/main/java/com/telenav/kivakit/core/vm/ShutdownHook.java

```diff
@@ -36,29 +37,52 @@
 @UmlClassDiagram(diagram = DiagramLanguage.class)
 public class ShutdownHook
 {
-    private static final LinkedList<ShutdownHook> queue = new LinkedList<>();
+    private static final int MILLISECONDS_TO_WAIT = 1200000;
+    private static final ShutdownHooks REGISTRY
+            = ShutdownHookRegistry.get(MILLISECONDS_TO_WAIT);
```

Comment:
Import ShutdownHookRegistry and change get() to take a Duration. Also consider changing get() method name to shutdownHookRegistry() to allow static importing to get rid of the class name. We do this in most places now for this reason.